internal/tiltfile/starkit/environment.go

```diff
@@ -146,31 +152,60 @@ func (e *Environment) start(path string) (Model, error) {
 }
 
 func (e *Environment) load(t *starlark.Thread, path string) (starlark.StringDict, error) {
-	return e.exec(t, AbsPath(t, path))
+	return e.exec(t, path)
 }
 
 func (e *Environment) exec(t *starlark.Thread, path string) (starlark.StringDict, error) {
-	// If the path isn't absolute, the loadCache won't work correctly.
-	if !filepath.IsAbs(path) {
-		return starlark.StringDict{}, fmt.Errorf("internal error: path must be absolute")
+	localPath, err := e.getPath(t, path)
+	if err != nil {
+		e.loadCache[localPath] = loadCacheEntry{
+			status:  loadStatusDone,
+			exports: starlark.StringDict{},
+			err:     err,
+		}
```

Comment:
i think you need to return here? can you add a test for the case where the interceptor returns an error?